OpenSubtitlesHandler/OpenSubtitles.cs

```diff
@@ -15,6 +15,8 @@ namespace OpenSubtitlesHandler
     /// </summary>
     public static class OpenSubtitles
     {
+        private static readonly CultureInfo _usCulture = CultureInfo.ReadOnly(new CultureInfo("en-US"));
```

Comment:
We're trying to move away from specific cultures. As far as I know Invariant is basically `en-US`, so this can go away

Reply:
Removed in ea3ebeb834ef7b86e31405f08b6237211f6c12e6